### PR TITLE
pep8 all py files

### PR DESCRIPTION
**Pull request by mufernando**
The [docs](https://stdpopsim.readthedocs.io/en/latest/development.html#coding-standards) say we follow the PEP8 guidelines but a quick run through `autopep8` reveals a bunch of inconsistencies.

Do we want to fix these?

### CODE REVIEW

**Comment by grahamgower** [comment]
Hey @mufernando, yes this looks sensible. Actually, the plan is to do this using the tool `black`, like tskit and msprime (see #433). But this would want simultaneous changes to (1) continuous integration scripts so that we check the formatting appropriately, and (2) updates to the dev docs.

**Comment by mufernando** [comment]
Can't we merge this and add `black` in the future? 

It's been such a pain for me to fix the issues with `lint` and I don't want to manually fix minor code format stuff all the time. Is there an alternative to this (that is not: remember all PEP8 guidelines when coding)?

**Comment by mufernando** [comment]
TL;DR: `tskit-dev` spoiled me with `black` in the pre-commit and now I don't want to manually fix formatting stuff.

**Comment by mufernando** [comment]
nvm, see #677
